docs/code/local-setup.md

```diff
@@ -11,8 +11,9 @@ Much of the software in this project is written in Python.
 It is usually worthwhile to install Python packages into a virtual environment,
 which allows them to be isolated from those in other projects which might have different version constraints.
 
-One popular solution for managing Python environments is [Anaconda/Miniconda](https://docs.conda.io/en/latest/miniconda.html).
-Another option is to use [`pyenv`](https://github.com/pyenv/pyenv).
+Most of our team uses [Anaconda](https://docs.anaconda.com/anaconda/install/).
+Another popular solution for managing Python environments is [Anaconda/Miniconda](https://docs.conda.io/en/latest/miniconda.html).
```

Comment:
This is confusing to me, since Miniconda is a variation of Anaconda, and you list Anaconda twice.